doc/source/methods/ALE.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<p>NB: This image does not appear in the rendered docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>It does...</p>